mclagsyncd/mclaglink.cpp

```diff
@@ -362,34 +538,32 @@ void MclagLink::setFdbEntry(char *msg, int msg_len)
     char *cur = NULL;
     short count = 0;
     int index = 0;
-    int exist = 0;
-    set <mclag_fdb>::iterator it;
 
-    cur = msg;
-    count = (short)(msg_len / sizeof(struct mclag_fdb_info));
+    cur = msg;           
```

Comment:
These extra spaces added/removed here and below seem to be unwanted and add more to the diff. Could you please cleanup them here and throughout the changes?